08-mapping.Rmd

```diff
@@ -53,24 +53,22 @@ Finally, Section \@ref(other-mapping-packages) covers a range of alternative map
-However, a decade later static plotting was still the emphasis of geographic data visualisation in R [@cheshire_spatial_2015].
-
-Despite the innovation of interactive mapping in R, static maps are still the foundation of mapping in R.
-The generic `plot()` function is often the fastest way to create static maps from vector and raster spatial objects, as shown in Sections \@ref(basic-map) and \@ref(basic-map-raster).
-Sometimes simplicity and speed are priorities, especially during the development phase of a project, and this is where `plot()` excels.
+Standard formats include `.png` and `.pdf` for raster and vector outputs respectively.
+Initially, static maps were the only type of maps that R could produce.
+Things have advanced greatly since the release of **sp** [see @pebesma_classes_2005] and many new techniques for map making have been developed.
+However, a decade later static plotting was still the emphasis of geographic data visualisation in R [@cheshire_spatial_2015] and still is its foundation despite the innovation of interactive mapping.
+
+The generic `plot()` function is often the fastest way to create static maps from vector and raster spatial objects (see sections \@ref(basic-map) and \@ref(basic-map-raster)).
+Sometimes, simplicity and speed are priorities, especially during the development phase of a project, and this is where `plot()` excels.
 The base R approach is also extensible, with `plot()` offering dozens of arguments.
-Another low-level approach is the **grid** package, which provides functions for low-level control of graphical outputs, --- see *R Graphics* [@murrell_r_2016], especially  Chapter [14](https://www.stat.auckland.ac.nz/~paul/RG2e/chapter14.html).
-The focus of this section, however, is making static maps with **tmap**, emphasizing the important aesthetic and layout options.
-
-Why **tmap**?
-It is a powerful and flexible map-making package with sensible defaults.
-It has a concise syntax that allows for the creation of attractive maps with minimal code, which will be familiar to **ggplot2** users.
-Furthermore, **tmap** has a unique capability to generate static and interactive maps using the same code via `tmap_mode()`.
-It accepts a wider range of spatial classes (including `raster` objects) than alternatives such as **ggplot2**, as documented in vignettes [`tmap-getstarted`](https://cran.r-project.org/web/packages/tmap/vignettes/tmap-getstarted.html) and [`tmap-changes-v2`](https://cran.r-project.org/web/packages/tmap/vignettes/tmap-changes-v2.html) and an academic paper on the subject [@tennekes_tmap_2018].
+Another low-level approach is the **grid** package which provides functions for basic control of graphical outputs---see *R Graphics* [@murrell_r_2016], especially  Chapter [14](https://www.stat.auckland.ac.nz/~paul/RG2e/chapter14.html). 
+This section however focuses on **tmap** and emphasizes the important aesthetic and layout options.
```

Comment:
Definite readability improvement. I would go further and remove the however if we make this change. I suggest changing to:

```
This section focuses on **tmap** and emphasizes the important aesthetic and layout options.
```